applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -4,3 +4,281 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
+#if (CONFIG_AUDIO_DEV == HEADSET)
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/audio/audio.h>
+
+#include "le_audio.h"
```

Comment:
"Local" includes (using "") under the "global" includes (using <>)

Reply:
We do usually keep the files own header file on top. So in streamctrl.c, the top header file is streamctrl.h, but macros_common and ctrl_events.h should be below the zephyr includes

Reply:
Still not correct in cis_gateway.
And to me concise: I have a newline after  #if (CONFIG_AUDIO_DEV == XXXXX) in the BIS files